test/remote/gateways/remote_pay_trace_test.rb

```diff
@@ -40,20 +40,12 @@ def test_acquire_token
   end
 
   def test_failed_access_token
```

Comment:
These three tests were effectively testing the same thing. combined them to one.